.github/workflows/ci.yml

```diff
@@ -12,7 +12,7 @@ env:
   # Common versions
   GO_VERSION: '1.19'
   GOLANGCI_VERSION: 'v1.49'
-  DOCKER_BUILDX_VERSION: 'v0.8.2'
+  DOCKER_BUILDX_VERSION: 'v0.10.0'
 
   # Common users. We can't run a step 'if secrets.AWS_USR != ""' but we can run
   # a step 'if env.AWS_USR' != ""', so we copy these to succinctly test whether
```

Comment:
Will need to add this for checks to work:
`UPBOUND_MARKETPLACE_PUSH_ROBOT_USR: ${{ secrets.UPBOUND_MARKETPLACE_PUSH_ROBOT_USR }}`